nova/virt/vmwareapi/vmops.py

```diff
@@ -2173,6 +2177,21 @@ def _resize_vm(self, context, instance, vm_ref, flavor, image_meta):
                                                     metadata=metadata)
         vm_util.reconfigure_vm(self._session, vm_ref, vm_resize_spec)
 
+        current_version = self._session._call_method(vutil,
+                                                     'get_object_property',
+                                                     vm_ref, 'config.version')
+
+        _current = self._hw_version_to_int(current_version)
+        if (vm_resize_spec.version and
+                _current < self._hw_version_to_int(
+                    vm_resize_spec.version)):
```

Comment:
(optional) I'd move that up and set `_current` and `_required` before the `if` to make the `if ` look nicer.